packages/rancher-istio-1.8/charts/templates/istio-uninstall-job.yaml

```diff
@@ -39,4 +39,7 @@ spec:
             name: istio-installer-overlay
         {{ end }}
       serviceAccountName: istio-installer
```

Comment:
So the uninstaller gets the installer PSP linked to it, we'll have to test both.

Reply:
Since its the same image, and the same service account then it makes sense to have one PSP for the installer and uninstaller. But I agree, we will need to test both install and uninstall to ensure all scripts run successfully.